Formula/json-table.rb

```diff
@@ -0,0 +1,15 @@
+class JsonTable < Formula
+  desc "Transform nested JSON data into tabular data in the shell"
+  homepage "https://github.com/micha/json-table"
+  url "https://github.com/micha/json-table/archive/4.3.3.tar.gz"
+  sha256 "0ab7bb2a705ad3399132060b30b32903762473ff79b5a6e6f52f086e507b0911"
+  head "https://github.com/micha/json-table.git"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "bash", "-c", "[ $(echo '{\"x\":{\"y\":[1,2,3]}}' | #{bin}/jt x y % | tail -n1) == 3 ]"
```

Comment:
This shouldn't need to use bash for the test.

Reply:
@ilovezfs the project has a `make test`, should I use that instead? Or just do something like `jt -h`

Reply:
`make test` isn't possible in the test block. I mean like
```
  test do
    output = pipe_output("#{bin}/jt x y %", '{"x":{"y":[1,2,3]}}', 0)
    assert_equal "3", output.lines.last.chomp
  end
```
though a test that's a bit more involved would be good. (See `jose` and `vim` for some examples)

Reply:
Ah, I see what you mean, thanks for clarifying. I incorporated your suggestion.